packages/components/src/components/flipcard/flipcard.ts

```diff
@@ -0,0 +1,277 @@
+import { css, html } from 'lit';
+import { customElement } from '../../internal/register-custom-element';
+import { property, query } from 'lit/decorators.js';
+import componentStyles from '../../styles/component.styles';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary Flipcard allows for the addition of content/information on both "sides" of the card, through means of a flip animation. Used  to add dynamism and interactivity to a page.
+ * @documentation https://solid.union-investment.com/[storybook-link]/flipcard
+ * @status stable
+ * @since 3.5.0
+ *
+ * @event sd-flip-front - Emmited when the front face of the flipcard is clicked.
+ * @event sd-flip-back - Emmited when the back face of the flipcard is clicked.
+ *
+ * @slot front - The front face of the flipcard.
+ * @slot back - The back face of the flipcard.
+ * @slot mediaFront - An optional media slot which can be as a background. Dependent from gradient variant.
+ * @slot mediaBack - An optional media slot which can be as a background. Dependent from gradient variant.
+ *
+ * @cssparts base - The component's base wrapper.
+ * @cssparts front - The container that wraps the front-side of the flipcard.
+ * @cssparts back - The container that wraps the back-side of the flipcard.
+ * @cssparts frontSlotContainer - The container that wraps the front slot.
+ * @cssparts backSlotContainer - The container that wraps the back slot.
+ * @cssparts mediaFront - The container that wraps the mediaFront slot.
+ * @cssparts mediaBack - The container that wraps the mediaBack slot.
+ * @cssparts frontSecondaryGradient - The container that wraps the secondary gradient of the front side.
+ * @cssparts backSecondaryGradient - The container that wraps the secondary gradient of the back side.
+ *
+ * @cssproperties --name - Description of the flipcard.
+ */
+@customElement('sd-flipcard')
+export default class SdFlipcard extends SolidElement {
+  @query('[part="back"]') back: HTMLElement;
+  @query('[part="front"]') front: HTMLElement;
+
+  /** Determines the variant of the front face of the flipcard. */
+  @property({ type: String, reflect: true })
+  frontVariant:
+    | 'primary'
+    | 'primary-100'
+    | 'gradient-light-top'
+    | 'gradient-light-bottom'
+    | 'gradient-dark-top'
+    | 'gradient-dark-bottom' = 'primary';
+
+  /** Determines the variant of the back face of the flipcard. */
+  @property({ type: String, reflect: true }) backVariant:
+    | 'primary'
+    | 'primary-100'
+    | 'gradient-light-top'
+    | 'gradient-light-bottom'
+    | 'gradient-dark-top'
+    | 'gradient-dark-bottom' = 'primary-100';
+
+  /** Determines the ratio for the layout of the flipcard. */
+  @property({ type: String, reflect: true }) ratio: '3:4' | '16:9' = '3:4';
+
+  connectedCallback() {
+    super.connectedCallback();
+  }
+
+  private flipFront() {
```

Comment:
From my perspective, the focus should switch to the back at the same time this function has been called. Currently the focus stays on the front element.

Reply:
I'm especially worried: If there are interactive elements – how should this work? So to make this accessible via keyboard, focus should stay on the whole flip card to be able to flip back immediately, right? But actually when the whole thing is tappable, I'm not sure if we can add nested things to tab as well?

Or should we maybe introduce a tabbable button which only shows up on focus?

This will be a tough one...

Reply:
I added the focus feedback on the latest iteration. Please take a look, especially on the sample to see how the flip-card behaves when there is tabbable content inside of the cards. @mariohamann @karlbaumhauer

Reply:
it works from my side even though I am still not happy with the hover... ;-) nevertheless, this should be decided by UX but for now I am fine.

Reply:
the flip-on-hover in general. But it might be a very personal oppinion.